docs/continuing_runs.md

```diff
@@ -0,0 +1,100 @@
+# Editing and extending runs
+
+The modularity of HADDOCK3 allows restarting halted runs from a certain step,
+extending previous runs with additional steps, and using successful steps as
+starting points for new runs. Let's see some examples.
+
+## Restarting a run
+
+Use the'--restart' flag to restart a HADDOCK3 run. For example, imagine you
+start a run:
+
+```
+haddock3 my-run-config.cfg
+```
+
+Unfortunately, something went wrong in step 4 (the fourth module defined in the
+configuration file).  You want now to continue the run from where it stopped.
+Use the `--restart` followed by `N`, where `N` is an integer corresponding to
+the step you which to restart from. **Remember**, haddock steps are 0-indexed
+(0, 1, 2, 3, ...); so, `--restart 3` is the fourth step.
+
+```
+haddock3 my-run-config.cfg --restart 3
+```
+
+## Restarting a run with modified parameters
```

Comment:
I would add here a comment that this will cause all steps >= N to be deleted from the current run